pkg/hs/urbit-king/lib/Urbit/Vere/Term/Logic.hs

```diff
@@ -111,8 +113,8 @@ drawState :: St -> [Ev]
 drawState St{..} = hist <> out <> cur <> spin
   where
     hist = drawHistory <$> toList sHistory
-    out  = if null sLine   then [] else [EvEdit sLine]
-    cur  = if 0 == sCurPos then [] else [EvMove $ fromIntegral $ sCurPos]
+    out  = if null sLine       then [] else [EvEdit sLine]
+    cur  = if 0 == fst sCurPos then [] else [EvMove sCurPos]
```

Comment:
Some people would say that you should use a guard here instead of an if expression, but I'll leave it up to you